### PR TITLE
Added autogrow on spyed css proprties change

### PR DESCRIPTION
**Pull request by nebarf**

### CODE REVIEW

File: angularjs-autogrow.js [hunk 1]
```diff
@@ -62,6 +62,30 @@
...
+                // Extract css properties to spy on
+                var spyProps = $attrs.autogrow.split(',');
+                angular.forEach(spyProps, function(property) {
+                  // Set a watcher on each property
+                  $scope.$watch(
+                    function() {
+                      return $element.css(property);
+                    },
+                    styleChangedCallBack,
+                    true
```
Comment (evyros) [inline angularjs-autogrow.js]: @frabnt looks great.
One question though: how necessary is the 3rd parameter of 'true' for deep watching?
Can you bring an example for it?

**Review by evyros — COMMENTED** [review]
#6 - watching css properties for changes.

File: angularjs-autogrow.js (hunk 1 above)
Comment (nebarf) [reply angularjs-autogrow.js]: Just checked angular doc and it's not needed for this use case. I'm going to update the PR

**Comment by astronati** [comment]
@evyros is the PR ok?

**Comment by evyros** [comment]
@astronati From a glimpse it looks good, I'll test it today before I release it officially.

**Comment by evyros** [comment]
Thanks @frabnt, that's exactly what was missing.
Please set the version number to 0.4.0 and I'll merge it.

**Comment by evyros** [comment]
Awesome! thank you for your contribution 👍

**Pull request merged**